ironic/tests/unit/drivers/modules/drac/test_raid.py

```diff
@@ -280,6 +280,53 @@ def test_clear_foreign_config_fail(self, mock_validate_job_queue,
             exception.DracOperationError, drac_raid.clear_foreign_config,
             self.node, 'RAID.Integrated.1-1')
 
+    @mock.patch.object(drac_job, 'validate_job_queue', spec_set=True,
+                       autospec=True)
+    def test_change_physical_disk_state(self,
+                                        mock_validate_job_queue,
+                                        mock_get_drac_client):
+        mock_client = mock.Mock()
+        mock_get_drac_client.return_value = mock_client
+
+        drac_raid.change_physical_disk_state(
+            self.node,
+            mode="JBOD",
+            controllers_to_physical_disk_ids=None)
+
+        mock_client.change_physical_disk_state.assert_called_once_with(
+            "JBOD", controllers_to_physical_disk_ids=None)
+
+    @mock.patch.object(drac_raid, 'change_physical_disk_state', spec_set=True,
+                       autospec=True)
+    @mock.patch.object(drac_raid, 'commit_config', spec_set=True,
+                       autospec=True)
+    def test__change_physical_disk_conversion(self,
+                                              mock_commit_config,
+                                              mock_change_physical_disk_state,
+                                              mock_get_drac_client):
+        mock_client = mock.Mock()
+        mock_get_drac_client.return_value = mock_client
+
+        expected_disk_state = mock_change_physical_disk_state.return_value = {
+            'is_reboot_required': True,
+            'conversion_results': {
+            'RAID.Integrated.1-1': {'is_reboot_required': 'optional',
+                                    'commit_required': False,
+                                    'is_commit_required': True}},
+            'commit_required_ids': ['RAID.Integrated.1-1'] }        
+        actual_disk_state = drac_raid.change_physical_disk_state(
```

Comment:
The line above really isn't executing anything becaused drac_raid.change_physical_disk_state is mocked above.  I think we should be calling _change_physical_disk_conversion() instead

Reply:
Yes, removed and replaced with actual method `_change_physical_disk_conversion()`